### PR TITLE
Design Tweaks

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/**](https://site-kyausyrrx.hackclub.dev/) |
|:-:|
|<a href="https://site-kyausyrrx.hackclub.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/DxL4Fixqr0MlxSLZsjuT8cMMyyNGrVVvffCj28UyoKRtVpEm4NW61GUVki-zwzxNwxjV9n46VKbq2wEY6TBDpQ.png">(view full size)</a> |







Commit 3b3653b2c6fa408865c723572d7b59f840474bfd (https://site-kyausyrrx.hackclub.dev).

**Comment by sampoder** [comment]
@bellesea mobile needs a bit of work but I'd love your thoughts!

**Comment by bellesea** [comment]
hii @sampoder sorry for the delay! thank you sm for pushing this, i love the changes and commented some nitpicks :)

File: pages/index.js [hunk 1]
```diff
@@ -810,36 +785,33 @@ function Page({
...
+                      color: 'orange'
+                    }}
+                  >
+                    IRL community.
+                  </Text>
+                  </Text>
+                <Text
+                  variant="subtitle"
+                  as="p"
+                  sx={{ fontSize: ['18px', '24px', '32px'], width: '66vw',margin: 'auto', pt: 2, pb: [3, 0, 0], textAlign: 'center' }}
```
Comment (sampoder) [inline pages/index.js]:
```suggestion
                  sx={{ fontSize: ['18px', '24px', '32px'], width: ['95vw', '66vw'],margin: 'auto', pt: 2, pb: [1, 0, 0], textAlign: 'center' }}
```

File: pages/index.js [hunk 2]
```diff
@@ -625,15 +603,15 @@ function Page({
...
                 </Text>{' '}
-                from around the world
-              </Text>
+                from around the world.
+                </Text>
               <Text
                 variant="subtitle"
                 as="p"
-                sx={{ fontSize: ['18px', '20px', '22px'], pb: [3, 0, 0] }}
+                sx={{ fontSize: ['18px', '24px', '32px'], width: '66vw',margin: 'auto', pt: 2, pb: [3, 0, 0], textAlign: 'center' }}
```
Comment (sampoder) [inline pages/index.js]:
```suggestion
                sx={{ fontSize: ['18px', '24px', '32px'], width: ['95vw','66vw'],margin: 'auto', pt: 2, pb: [1, 0, 0], textAlign: 'center' }}
```

File: pages/index.js [hunk 3]
```diff
@@ -810,36 +785,33 @@ function Page({
...
+                      color: 'orange'
+                    }}
+                  >
+                    IRL community.
+                  </Text>
+                  </Text>
+                <Text
+                  variant="subtitle"
+                  as="p"
+                  sx={{ fontSize: ['18px', '24px', '32px'], width: ['95vw', '66vw'],margin: 'auto', pt: 2, pb: [1, 0, 0], textAlign: 'center' }}
```
Comment (sampoder) [inline pages/index.js]:
```suggestion
                  sx={{ fontSize: ['18px', '24px', '32px'], width: ['100%', '66vw'],margin: 'auto', pt: 2, textAlign: 'center' }}
```

File: components/index/cards/slack.js [hunk 4]
```diff
@@ -15,18 +15,18 @@ const Cover = () => (
...
   />
 )
-const Stats = ({ data, subheading }) => (
+const Stats = ({ data, subheading, nonMobile = false }) => (
   <Box>
     <Heading
       variant="headline"
       as="h4"
-      sx={{ mb: 0, pt: 2, fontSize: ['28px', '36px', '38px'] }}
+      sx={{ mb: 0, pt: 2, fontSize: ['28px', '36px', '38px'], display: nonMobile ? ['none', 'block'] : 'block' }}
```
Comment (sampoder) [inline components/index/cards/slack.js]:
```suggestion
      sx={{ mb: 0, pt: 2, fontSize: ['28px', '36px', '38px'] }}
```

File: components/index/cards/slack.js [hunk 5]
```diff
@@ -15,18 +15,18 @@ const Cover = () => (
...
-        'linear-gradient(to bottom,rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0.85))',
+        'linear-gradient(to bottom,rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.95))',
       opacity: 0.8,
       zIndex: 1
     }}
   />
 )
-const Stats = ({ data, subheading }) => (
+const Stats = ({ data, subheading, nonMobile = false }) => (
   <Box>
```
Comment (sampoder) [inline components/index/cards/slack.js]:
```suggestion
  <Box sx={{ display: nonMobile ? ['none', 'block'] : 'block' }}>
```

**Pull request merged**